newlib/libc/machine/riscv/memcpy-asm.S

```diff
@@ -14,14 +14,14 @@
 .global memcpy
 .type	memcpy, @function
 memcpy:
-  mv t1, a0
+  mv a3, a0
   beqz a2, 2f
 
 1:
-  lb t2, 0(a1)
-  sb t2, 0(t1)
+  lbu a4, 0(a1)
```

Comment:
good catch. this does not really change the function, but lb would include some unnecessary sign extension operation

Reply:
make the change from lb to lbu a separate commit. It is atomic, and this allows you to clearly give the rationale for the change in the commit message.